Projects/App/Sources/UI/Place/PlaceList/View/Cell/PlaceCell.swift

```diff
@@ -0,0 +1,166 @@
+    private lazy var menuLabel = UILabel()
+    
+    private lazy var bottomView = UIView()
+    
+    private lazy var placeNameLabel = UILabel()
+    
+    private lazy var emojiStackView = UIStackView()
+    
+    lazy var goodEmojiStackView = EmojiCountStackView(emojiCount: viewModel.goodCount, emoji: UIImage(.img_reviewfriends_good_30)!)
+    lazy var sosoEmojiStackView = EmojiCountStackView(emojiCount: viewModel.sosoCount, emoji: UIImage(.img_reviewfriends_soso_30)!)
+lazy var badEmojiStackView = EmojiCountStackView(emojiCount: viewModel.badCount, emoji: UIImage(.img_reviewfriends_bad_30)!)
+    
+    // MARK: - LifeCycle
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        createLayout()
+        configureUI()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+}
+
+// MARK: - UI Function
+
+extension PlaceCell {
+    
+    func configureUI() {
+        configureGradientView()
+        
+        mainStackView.spacing = 0
+        mainStackView.axis = .vertical
+        mainStackView.distribution = .fill
+        mainStackView.layer.cornerRadius = 16
+        mainStackView.layer.masksToBounds = true
+        
+        if !viewModel.isReviewExists {
+            mainStackView.layer.borderWidth = 1
+        }
+        
+        reviewImageView.image = viewModel.reviewImage
+        reviewImageView.contentMode = .scaleAspectFit
+        
+        menuLabel.text = viewModel.menuName
+        menuLabel.textColor = .white
+        
+        bottomView.backgroundColor = .label
+        
+        placeNameLabel.textColor = .zestyColor(.whiteEBEBF5)
+        placeNameLabel.text = viewModel.placeName
+        placeNameLabel.numberOfLines = 2
+        placeNameLabel.font = .systemFont(ofSize: 20, weight: .bold)
+        
+        emojiStackView.spacing = 12
+        emojiStackView.axis = .horizontal
+        emojiStackView.distribution = .fill
+    }
+    
+    private func createLayout() {
+        contentView.addSubview(mainStackView)
+        mainStackView.addArrangedSubviews([reviewImageView, middelView, bottomView])
+        middelView.addSubviews([gradientView, menuLabel])
+        bottomView.addSubviews([placeNameLabel, emojiStackView])
+        
+        emojiStackView.addArrangedSubviews([goodEmojiStackView, sosoEmojiStackView, badEmojiStackView])
+        
+        mainStackView.snp.makeConstraints { make in
+            make.top.equalToSuperview().inset(10)
+            make.horizontalEdges.equalToSuperview().inset(horizontalPadding)
+            make.bottom.equalToSuperview().inset(20)
+        }
+        
+        reviewImageView.snp.makeConstraints { make in
+            make.top.equalToSuperview()
+            make.horizontalEdges.equalToSuperview()
+            make.height.equalTo(screenWidth - horizontalPadding * 2)
```

Comment:
p4;

뭔가 107번째 줄
높이 = 너비를 의도하신 것 같은데 그런 의도라면 요렇게 작성하는 것을 추천드립니다!


```suggestion
            make.height.equalTo(reviewImageView.snp.width)
        }
```

Reply:
이런 방법이 있었네요!
변경했습니다! da34967acb00d3f040eaebb1bb3c5f726c4f6efc